pynicotine/search.py

```diff
@@ -451,14 +452,69 @@ def create_search_result_list(self, searchterm, wordindex, excluded_words, parti
             log.add_debug("Error: DB closed during search, perhaps due to rescanning shares or closing the application")
             return None
 
-    def process_search_request(self, searchterm, username, token, direct=False):
+    def create_file_info_list(self, results, max_results, permission_level):
```

Comment:
`create_file_info_list` -> `_create_file_info_list`

This method wouldn't be any use from any other module without `results`, so it might as well be private.